content/news/2018-05-30-developer-content-manifestos-retesting-navcoin-4-2-0.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: news
+generated_by_cms: true
+title: 'Developer & Content Manifestos, Retesting NavCoin 4.2.0'
+author: Craig MacGregor
+date: '2018-05-30T17:44:23+12:00'
+feature_image: /images/uploads/may-20.png
+categories:
+  - Nav News
+---
+Following on from the NavCoin Brand Guidelines published a couple of weeks ago, the NavCoin Core content contributors have added even more resourceful content that will further help empower community members to become active contributors in their own right. Expanding on that theme, there has been plenty of activity by community members this past week - from more merchants in Italy to a tipping bot on Telegram. Developers have been hard at it too, focussing on the retesting of the NavCoin Core 4.2.0 release candidate, and creation of NavCoin Protocol Improvement Proposals (NPIP’s).
+
+<!--more-->
+
+## What’s happening in the community
+
+We noticed some questions popping up in the community recently about Staking, especially the differences between Proof of Stake and Proof of Work. Here’s a great guide from Bocyaj that goes into detail about how it all works.
+
+<https://medium.com/@bocyaj2012/nav-coin-proof-of-stake-informational-guide-71c9c3417212>
+
+NavCoin Italia has continued his efforts to make Italy the NavCoin merchant centre of the universe. This week they’ve added a furniture store and a fast food restaurant to the list.
+
+![NavCoin Italia Merchant Map](/images/uploads/screen-shot-2018-05-29-at-8.38.45-am.png)
+
+On Reddit, Jonathan reports that he’s considering starting a community Twitter account. He hopes that adding another communication channel will encourage better networking among members. Nice one Jonathan.
```

Comment:
not quite, I want to collect existing accounts from the community in a comprehensive list in order to boost the communities reach

Reply:
updated

Reply:
"On Reddit, Jonathan reports he wants to collect existing accounts from the community in a comprehensive list in order to boost the community's reach. If you run or know of any NavCoin social channels, head on over to the thread and let him know. Nice one Jonathan."